assembl/static2/js/app/components/debate/survey/question.jsx

```diff
@@ -140,7 +136,9 @@ class Question extends React.Component {
                   }}
                   value={this.state.postBody}
                   maxLength={300}
-                  ref={(t) => { this.txtarea = t; }}
+                  ref={(t) => {
+                    this.txtarea = t;
```

Comment:
This ref object, it is a DOM object yes? Or is a React simulated? I see that above it's used as `this.txtarea.props.maxLength`. Wasn't aware that a textbox had props property